scripts/run_integration_tests.sh

```diff
@@ -0,0 +1,109 @@
+#!/bin/bash
+
+# Usage - run commands from repo root:
+# To check if new changes to the library cause changes to any snapshots:
+#   ./scripts/run_integration_tests.sh
+# To regenerate snapshots:
+#   UPDATE_SNAPSHOTS=true ./scripts/run_integration_tests.sh
+
+set -e
+
+# To add new tests create a new ts file in the 'integration_tests/stacks' directory, append its file name to the STACK_CONFIGS array.
+# Note: Each ts file will have its respective snapshot built in the snapshots directory, e.g. lambda-function-stack.ts
+#       will generate both snapshots/test-lambda-function-stack-snapshot.json and snapshots/correct-lambda-function-stack-snapshot.json
+STACK_CONFIGS=("lambda-function-stack" "lambda-nodejs-function-stack")
+
+script_path=${BASH_SOURCE[0]}
+scripts_dir=$(dirname $script_path)
+repo_dir=$(dirname $scripts_dir)
+root_dir=$(pwd)
+if [[ "$root_dir" =~ .*"datadog-cdk-constructs/scripts".* ]]; then
+    echo "Make sure to run this script from the root $(datadog-cdk-constructs) directory, aborting"
+    exit 1
+fi
+
+integration_tests_dir="$repo_dir/integration_tests/"
+if [ "$UPDATE_SNAPSHOTS" = "true" ]; then
+    echo "Overwriting snapshots in this execution"
+fi
+
+# Login to ECR. This is a required step in order to pull a public Docker image for the PythonFunction construct
+aws ecr-public get-login-password --region us-east-1 | docker login --username AWS --password-stdin public.ecr.aws
+
+yarn
+yarn build
+
+cd $integration_tests_dir
+
+OUTPUT_ARRAY+=("====================================")
+allTestsPassed=true
```

Comment:
By convention, variables in Bash scripts are `ALL_CAPS_AND_SNAKE_CASED`. There are a bunch of variables in this file with different capitalization schemes, let's update them all to be consistent.